app/Http/Controllers/V2/Dashboard/ActiveCountriesTableController.php

```diff
@@ -33,25 +33,27 @@ public function getAllCountries()
             ->get();
         $activeCountries = [];
         foreach ($countries as $country) {
-            $totalProjects = $this->numberOfProjects($country->slug, $projects);
-
-            $totalSpeciesAmount = $this->totalSpeciesAmount($country->slug, $projects);
-
-            $totalJobsCreated = $this->totalJobsCreated($country->slug, $projects);
-
-            $numberOfSites = $this->numberOfSites($country->slug, $projects);
-
-            $totalNurseries = $this->numberOfNurseries($country->slug, $projects);
-
-            $activeCountries[] = [
-                'country_slug' => $country->slug,
-                'country' => $country->label,
-                'number_of_projects' => $totalProjects,
-                'total_trees_planted' => $totalSpeciesAmount,
-                'total_jobs_created' => $totalJobsCreated,
-                'number_of_sites' => $numberOfSites,
-                'number_of_nurseries' => $totalNurseries,
-            ];
+            if ($this->numberOfProjects($country->slug, $projects) > 0) {
```

Comment:
Store this value `$this->numberOfProjects($country->slug, $projects)` in a variable and use it for conditional